styles/components/App.less

```diff
@@ -30,12 +29,12 @@
   margin-left: auto;
   margin-right: auto;
   margin-bottom: 20px;
-  min-height: 100px;
-  max-height: 420px;
+  min-height: 470px;
```

Comment:
I think there's misunderstanding here :/

I'll try to illustrate (from add-omnipod-select branch on origin). Here's what the UI looks like for someone who has multiple people they are allowed to upload to (so the groups dropdown is present) and uses a Medtronic pump, a Dexcom, and the PX meter (i.e., a likely Jaeb study participant).

![screenshot 2015-02-19 17 51 53](https://cloud.githubusercontent.com/assets/1588547/6280451/080b3ae8-b862-11e4-8b20-a9e3a640f869.png)

The `max-height` I chose for the inner scrolling container was because of this case - I wanted to make sure that for such a person two devices are visible in their entirety, and they can scroll to see the third, but the third isn't cut off.

I neglected to notice, however, that there is still wasted space in the footer. If I can paraphrase @HowardLook's feedback, I think it was that we'd like to have the footer closer to the bottom of the app (and in the same place no matter where in the app you are - location in the app is what I actually meant by "screens") and try to increase the space given to the scrolling content. It would be great if three devices could fit in the upload list without engaging scrolling (maybe only possible for users that don't have permissions to upload to anyone else, so the groups dropdown isn't present).

Also FYI, I believe the decision to have two logos present was because with only the T logo there is no company identification anywhere in the app.